ansible/roles/pg_standby/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+# roles/pg_standby/tasks/main.yml
+
+- name: Create wal archive directory
+  sudo: yes
+  file:
+    path: '{{ archive_dir }}'
+    owner: postgres
+    group: postgres
+    mode: 0755
+    state: directory
+
+- name: PostgreSQL recovery configuration
+  sudo_user: postgres
+  template: src=recovery.conf.j2 dest={{ postgresql_config_home }}/recovery.conf
+  notify: Restart postgres
+
+- name: create backup directory
+  sudo: yes
+  file:
+    path: '{{ postgresql_backup_dir }}'
+    owner: postgres
+    group: postgres
+    mode: 0755
+    state: directory
+  when: backup_postgres
+
+- name: Copy postgres backup script
+  sudo: yes
+  template:
+    src: "create_postgres_dump.sh.j2"
+    dest: "/etc/cron.d/create_postgres_dump.sh"
+    group: postgres
+    owner: postgres
+    mode: 0700
+    backup: yes
+  when: backup_postgres
```

Comment:
ah i see that that module doesn't let you do templating and that you already found the cron module below